pkg/engine/plan.go

```diff
@@ -183,6 +183,7 @@ func (planResult *planResult) Walk(cancelCtx *Context, events deploy.Events, pre
 			RefreshOnly:       planResult.Options.isRefresh,
 			RefreshTargets:    planResult.Options.RefreshTargets,
 			DestroyTargets:    planResult.Options.DestroyTargets,
+			UpdateTargets:     planResult.Options.UpdateTargets,
```

Comment:
@pgavlin thinking about merging these three 'targets' into one single array.  Thoughts on if you'd prefer that over the current approach?